### ISSUE TITLE
Default parameter used

### ISSUE THREAD

**Issue by pietroastolfi** [hkust-vgd/scanobjectnn#8]
First Q: do the default parameters indicated correspond to the ones actually used for the experiments reported in the paper? 

Second Q:  In the code of PointNet train sometimes you have the `default` field of argparse different from the default suggested by the help of the same argument. Which one should be trusted?

**Comment by mikacuy** [comment]
Hi,

1: Yes the defaults are those used in the experiments. Double check that --norm=True

2: The default field should be used.

Best,
Mika

**Comment by pietroastolfi** [comment]
Hi Mika,

I cannot repeat the results of pointnet for the dataset PB_50_TS with BG, main_split. Using the pn model with the two T-net I get no improvement w.r.t the basic model without them, do you confirm? 
Furthermore, the use of centered and normalized point-cloud decreases the performance. I copied the code you use for center and norm, so I do not expect bugs over there.

**Comment by mikacuy** [comment]
Hi,

You can ignore the --center_data flag, that was used for another set of experiments before I created the .h5 files. When using the h5 files, please ignore that flag.

--norm=True was used in all experiments, it basically makes all the objects of the same scale. It is natural that the performance drops when this flag is true because the is no information on size. (eg a cabinet vs a box would be harder to classify. All experiments reported in the paper used --norm=True

Training data augmentation (random rotation wrt to the up direction) is used in all our experiments (for all models). If I'm not mistaken, all the default codes of the original implementations had this type of augmentation.

I did not try removing the T-net from the pointnet model. In the paper, we do not try to argue for rotation invariance of the networks, so we just use the models with their augmentations as they were previously implemented.

Best,
Mika
